### ISSUE TITLE
Delete obsolete `codexia_review_notifications`

### ISSUE THREAD

**Issue by iakunin** [iakunin/codexia-bot#186]
- [ ] Delete obsolete `codexia_review_notifications` (all except the latest one)
- [ ] Run PG vacuum

**Comment by iakunin** [comment]
Before cleaning:
```
select count(id) from codexia_review_notification;
218054
```

After cleaning:
```
select count(id) from codexia_review_notification;

```

**Comment by 0pdd** [comment]
@iakunin 2 puzzles [#198](https://github.com/iakunin/codexia-bot/issues/198), [#199](https://github.com/iakunin/codexia-bot/issues/199) are still not solved.

**Comment by 0pdd** [comment]
@iakunin the puzzle [#199](https://github.com/iakunin/codexia-bot/issues/199) is still not solved; solved: [#198](https://github.com/iakunin/codexia-bot/issues/198).

**Comment by 0pdd** [comment]
@iakunin all 2 puzzles are solved here: [#198](https://github.com/iakunin/codexia-bot/issues/198), [#199](https://github.com/iakunin/codexia-bot/issues/199).
